mujoco_warp/_src/passive.py

```diff
@@ -286,6 +286,80 @@ def _qfrc_passive(
   qfrc_passive_out[worldid, dofid] = qfrc_passive
 
 
+@wp.kernel
+def _flex_elasticity(
+  # Model:
+  body_dofadr: wp.array(dtype=int),
+  flex_dim: wp.array(dtype=int),
+  flex_vertadr: wp.array(dtype=int),
+  flex_edgeadr: wp.array(dtype=int),
+  flex_elemedgeadr: wp.array(dtype=int),
+  flex_vertbodyid: wp.array(dtype=int),
+  flex_elem: wp.array(dtype=int),
+  flex_elemedge: wp.array(dtype=int),
+  flexedge_length0: wp.array(dtype=float),
+  flex_stiffness: wp.array(dtype=float),
+  flex_damping: wp.array(dtype=float),
+  # Data in:
+  flexvert_xpos_in: wp.array2d(dtype=wp.vec3),
+  flexedge_length_in: wp.array2d(dtype=float),
+  flexedge_velocity_in: wp.array2d(dtype=float),
+  # In:
+  timestep: float,
```

Comment:
can we update `timestep` -> `opt_timestep` and move this input under `#Model:` in order to match the convention in the repository?

for example
https://github.com/google-deepmind/mujoco_warp/blob/12256ba37842738d8684d44c611cb659cda28930/mujoco_warp/_src/forward.py#L57

Reply:
Done.